riotfile.py

```diff
@@ -1159,5 +1159,16 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ]
             },
         ),
+        Venv(
+            name="aredis",
+            pys=select_pys("3.6"),
```

Comment:
```suggestion
            pys=select_pys(min_version="3.6"),
```

Makes it a little easier to read/scan.